media/static-bundles.json

```diff
@@ -1072,6 +1074,15 @@
       ],
       "name": "firefox_new_scene1"
     },
+    {
+      "files": [
+        "js/base/mozilla-fxa.js",
```

Comment:
Perhaps not... I thought it was otherwise required for the form to work but on further inspection all it's doing it adding a class as a style hook, which I'm not even using, so it can go.